git-paste: add page (#10992)
tldr-pages/tldr: pages/common/git-paste.md

# git paste

> > Send commits to a pastebin site using pastebinit.

Reviewer: ```suggestion
> Send commits to a pastebin site using `pastebinit`.
```

> Part of `git-extras`.
> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-paste>.

> - Send commits to a pastebin site using pastebinit:

Reviewer: ```suggestion
- Send commits to a pastebin site using `pastebinit`:
```

> - Send commits to a pastebin site using `pastebinit`:
> 
> `git paste {{paste_url}}`

Reviewer 2: ```suggestion
- Send the patches between the current branch and its upstream to a pastebin using `pastebinit`:

`git paste`
```

> - You can also pass options understood by git-rev-parse(1) in order to select a different set of commits:

Reviewer: ```suggestion
- Pass options understood by git-rev-parse(1) in order to select a different set of commits:
```

> - Pass options understood by `git-rev-parse(1)` in order to select a different set of commits:
> 
> `git paste @^ {{paste_url}}`

Reviewer 2: ```suggestion
- Pass options to `git format-patch` in order to select a different set of commits (`@^` selects the parent of HEAD, and so the currently checked out commit is sent):

`git paste {{@^}}`
```
